modules/nf-core/cellranger/multi/main.nf

```diff
@@ -44,23 +44,24 @@ process CELLRANGER_MULTI {
 
     // if references + FASTQ are empty, then don't run corresponding analyses
     // get names of references, if they exist
-    // empty reference channels stage as "references"
+    // empty reference channels can stage as "[]" when skipped by the workflow, or get an 'EMPTY' file when empty
```

Comment:
Any point in retaining the `EMPTY` if `[]` is now supported?

Reply:
No particular reason. I just did not try yet. So was retained as 'legacy' compatibility.

But this is a good point.
We can solve this and keep entirely as '[]' instead of the EMPTY file.

But it needs a proper testing to guarantee these are the only points it is used.

Reply:
Hi @grst ,
I have just updated it to not rely on `EMPTY` file anymore. It works entirely with `[]`.
You can see in the `nf-test` that I modified it to use `[]` and did not change anything in the snaps, being all the same. And still passing all tests.